lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -42,6 +42,12 @@ function subscribeEvents() {
   });
 }
 
+export function sendEvent(category, action, label) {
+  if (typeof ga !== 'undefined' && ga !== null) {
```

Comment:
Is this conditional still needed? Aren't you already using it in the `init()` which calls this function? or is it because you're exporting this function?